apps/search-poc/src/components/DashboardList.tsx

```diff
@@ -8,18 +8,8 @@ export type DashboardItem = {
 const DashboardList: React.FunctionComponent<{
   items: DashboardItem[];
   onDashboardSelected: (id: string) => void;
-  defaultActiveId?: string;
-}> = ({ items, onDashboardSelected, defaultActiveId }) => {
-  const [activeDashboardId, setActiveDashboardId] = React.useState<string>(
-    defaultActiveId || items[0].id,
-  );
-
-  React.useEffect(() => {
-    if (activeDashboardId !== null) {
-      onDashboardSelected(activeDashboardId);
-    }
-  }, [activeDashboardId, onDashboardSelected]);
```

Comment:
is you remove `onDashboardSelected` it will fix your history issue (but it will a cause a lint error)

Reply:
I did it woo